src/components/game/GameDetails.js

```diff
@@ -0,0 +1,89 @@
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import Button from 'material-ui/Button'
+import Card, { CardActions, CardContent } from 'material-ui/Card'
+import Typography from 'material-ui/Typography'
+import {withRouter} from 'react-router'
+import {joinGame} from '../../actions/games'
+import './GameDetails.css'
+
+class GameDetails extends PureComponent {
+
+  handleClick = () => {
+    const {game,history,players} = this.props
+    switch(this.buttonText(players)) {
+      case "join":
+        this.props.joinGame(game.id)
+        history.push(`/games/${game.id}`)
+        break
+      case "enter":
+      case "watch":
+        history.push(`/games/${game.id}`)
+        break
+      default:
+        return null
+    }
+  }
+  buttonText = (players) => {
+    const id = this.props.userId
+    const game = this.props.game
+    if (players.includes(id)) {
+      if (game.status === "started") return "enter"
+      return "..."
+    } else {
+      if (game.status === 'started') return "watch"
+      return "join"
+    }
+  }
+  render() {
+    const {game,users,userId, players} = this.props
+    //const playerIds = game.players.map(player => player.userId)
+
+    if (game.status==="finished") return null
+
+    return (
+      <Card className="game-card">
+        <CardContent>
+          <Typography color="textSecondary">
```

Comment:
what's the <Typography> tag? 
and the Cards XD ?

Reply:
I don't know, You put that right? haha. I just copied them :smile_cat: 
These are from material_ui but I'm not even sure which part of the render is that lol

Reply:
oh omg of course the material_ui stuff. I already forgot how that works again.. weird stuff XD I think I do like CSS haha